src/main/java/dk/kb/util/BuildInfoManager.java

```diff
@@ -85,6 +90,45 @@ public static String getBuildTime() {
         return buildTime;
     }
 
+    /**
```

Comment:
The `BuildInfoManager` needs to be extended to **optionally** take the filename of the build properties as argument, as applications should use custom filenames for it to avoid confusion as to where a build properties file belongs.